manticore/native/cpu/abstractcpu.py

```diff
@@ -521,6 +530,7 @@ def __init__(self, regfile: RegisterFile, memory: Memory, **kwargs):
         self._instruction_cache: Dict[int, Any] = {}
         self._icount = 0
         self._last_pc = None
+        self._last_executed_pc = None
```

Comment:
This is slightly different than `last_pc` and does indeed capture the last executed instruction address, whereas I had some issues with `last_pc` when trying to use it for that purpose.

Reply:
Opened PR https://github.com/trailofbits/manticore/pull/2475